services/catalog/src/simcore_service_catalog/api/routes/services_resources.py

```diff
@@ -150,7 +150,7 @@ async def _get_service_labels(
         # and will fail validating the key or the version
         if err.status_code == status.HTTP_400_BAD_REQUEST:
             return None
-        raise err
+        raise
```

Comment:
For my learning: What's actually the difference between `raise err` and `raise` here?

Reply:
In this context it is the same. 
`raise` alone is a short way of saying "re-raise the exception caught in this context"